features/wasm-bigint.yml

```
@@ -0,0 +1,6 @@
+name: WebAssembly BigInt to i64 conversion in JavaScript
```

You'll see some more comments along these lines elsewhere, but I'd prefer that we do not prefix every wasm feature. Instead, we can either do things like this (if there's no ambiguity)….

```suggestion
name: BigInt to i64 conversion
```

Or append the disambiguation:

```suggestion
name: WebAssembly BigInt to i64 conversion in JavaScript (WebAssembly)
```